packages/cardpay-sdk/sdk/safes/base.ts

```diff
@@ -200,36 +200,50 @@ export default class Safes {
     return gasInToken;
   }
 
+  async sendTokens(txnHash: string): Promise<TransactionReceipt>;
   async sendTokens(
     safeAddress: string,
     tokenAddress: string,
     recipient: string,
     amount: string,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async sendTokens(
+    safeAddressOrTxnHash: string,
+    tokenAddress?: string,
+    recipient?: string,
+    amount?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    if (!tokenAddress) {
```

Comment:
ditto with the comments above about testing whether the value you actually care about is shaped like an address or txnHash instead of testing a side effect